static/app/views/settings/organizationTeams/allTeamsRow.tsx

```diff
@@ -199,6 +213,17 @@ class AllTeamsRow extends Component<Props, State> {
     const canViewTeam = team.hasAccess;
 
     const idpProvisioned = team.flags['idp:provisioned'];
+    const teamOrgRole = team.orgRole
+      ? team.orgRole.charAt(0).toUpperCase() + team.orgRole.slice(1) + ' Team'
```

Comment:
1. We have `lodash` as a dependency so you can use [startCase](https://lodash.com/docs/4.17.15#startCase) to avoid charAt(0) and slice()
2. You can use [string templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to avoid doing 'my string' + ' another string'